qemu/qemu.go

```diff
@@ -995,28 +963,6 @@ func (vsock VSOCKDevice) Valid() bool {
 	return true
 }
 
-// QemuParams returns the qemu parameters built out of the VSOCK device.
-func (vsock VSOCKDevice) QemuParams(config *Config) []string {
-	var deviceParams []string
-	var qemuParams []string
-
-	deviceParams = append(deviceParams, fmt.Sprintf("%s", VhostVSOCKPCI))
```

Comment:
I guess the change was `VhostVSOCKPCI` to `VhostVSOCK` right?, if that's correct could you please just change this line and not all the function, since it's not easy to read and detect the change